beacon-chain/core/peerdas/helpers.go

```diff
@@ -290,12 +274,8 @@ func VerifyDataColumnSidecarKZGProofs(sc *ethpb.DataColumnSidecar) (bool, error)
 		ckzgComms = append(ckzgComms, kzg.Bytes48(com))
 	}
 	var cells []kzg.Cell
-	for _, ce := range sc.DataColumn {
-		var newCell []kzg.Bytes32
-		for i := 0; i < len(ce); i += 32 {
-			newCell = append(newCell, kzg.Bytes32(ce[i:i+32]))
-		}
-		cells = append(cells, kzg.Cell(newCell))
```

Comment:
same as https://github.com/prysmaticlabs/prysm/pull/14159/files#r1664747041

We (the caller) no longer needs to convert each field element in the cell into a bytes32 array